app/assets/stylesheets/application.css

```diff
@@ -14,16 +14,15 @@
  *= require_self
  */
 
+@import "components/banner-small";
 @import "components/button-holder";
 @import "components/buttons";
+@import "components/content";
 @import "components/footer";
 @import "components/navbar";
 @import "config/base";
+@import "config/colors";
 @import "config/fonts";
 @import "config/rotations";
 @import "config/spaces";
 @import "pages/homepage.css";
```

Comment:
**nitpick**: I usually prefer to organize imports from less specific to more specific in order to enact proper cascading. In this case we'd start from config, to components and then to pages.